yaml/shared/src/main/scala/zio/config/yaml/YamlConfigSource.scala

```diff
@@ -108,7 +108,7 @@ object YamlConfigSource {
     Try {
```

Comment:
Unrelated. Sooner, with the merge of https://github.com/zio/zio-config/pull/636 and https://github.com/zio/zio-config/pull/635, we don't need this Try. But I guess I will wait for this to be merged before those goes in.